### PR TITLE
Add version package and flag

### PR DESCRIPTION
**Pull request by bgilbert**
Fixes coreos/bugs#1741.

### CODE REVIEW

**Comment by bgilbert** [comment]
Sure.  Ignition assumes that it's always built from Git, not from a tarball; is that assumption safe here?

**Comment by crawford** [comment]
Assuming this will be built from a git clone is safe. We'll want to add instructions for building this to the README (git clone, git checkout tag, ./build)

File: README.md [hunk 1]
```diff
@@ -4,3 +4,15 @@ The config transpiler ("ct" for short) is the utility responsible for transformi
 
 [ignition]: https://github.com/coreos/ignition
 [overview]: doc/overview.md
+
+## Building
+
+```shell
+git clone https://github.com/coreos/container-linux-config-transpiler
```
Comment (crawford) [inline README.md]: How about `git clone --branch v0.1.0 https://github.com/coreos/container-linux-config-transpiler`?

File: internal/main.go [hunk 2]
```diff
@@ -50,6 +53,11 @@ func main() {
 		return
 	}
 
+	if flags.version {
+		fmt.Printf("%s\n", version.String)
```
Comment (crawford) [inline internal/main.go]:
```go
fmt.Printf("ct %s\n", version.String)
```

Comment (bgilbert) [inline internal/main.go]: Following the Ignition example, I put `ct` inside `version.String`.  I can move it though.

Comment (crawford) [inline internal/main.go]: Ah, I missed that. No need to use a formatting print then.

**Comment by crawford** [comment]
Looks good. Can you update the commit messages to match the rest of the history?

**Pull request merged**